js/ts_global.d.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+// This scopes the `ts` namespace globally, which is where it exists at runtime
+// when building Deno, but the `typescript/lib/typescript.d.ts` is defined as a
+// module.
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Is this `eslint-disable-next-line @typescript-eslint/no-unused-vars` necessary? Seems like it's used...

Reply:
changed to `ts_` but the eslint error persists for some reason... I think it maybe a bug with ESLint, in that it isn't tracking the usage below.  I don't know exactly why.  I will open an issue with `@typescript-eslint`.